src/rest-server/src/models/v2/utils/frameworkConverter.js

```diff
@@ -379,7 +466,145 @@ const convertTaskDetail = async (
   };
 };
 
+const convertTaskAttempt = async (
+  logPathInfix,
```

Comment:
What is the difference between taskStatus.attemptStatus and attemptStatus  here?

Reply:
Add comment here

---
In reply to: [506201437](https://github.com/microsoft/pai/pull/4958#discussion_r506201437) [](ancestors = 506201437)

Reply:
> What is the difference between taskStatus.attemptStatus and attemptStatus here?
Changed the order of function parameters and added comments.